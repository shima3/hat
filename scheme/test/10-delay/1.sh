#!/bin/bash
extless="${0%.*}"
$* examples/delay.sch < ${extless}.in
