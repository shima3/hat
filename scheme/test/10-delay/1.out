Start
aho
barker
aho
barker
End
